server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -197,16 +196,18 @@ public RemoteClusterStateService(
         this.remoteStateStats = new RemotePersistenceStats();
         this.namedWriteableRegistry = namedWriteableRegistry;
         this.indexMetadataUploadListeners = indexMetadataUploadListeners;
+        this.clusterService = clusterService;
```

Comment:
Do we need this as a class variable now that we are creating service here only?

Reply:
Removed.